--- desk.orig	2020-05-03 12:50:51.650500668 +0200
+++ desk	2020-05-03 13:01:46.253969063 +0200
@@ -4,6 +4,7 @@
 PREFIX="${DESK_DIR:-$HOME/.desk}"
 DESKS="${DESK_DESKS_DIR:-$PREFIX/desks}"
 DESKFILE_NAME=Deskfile
+DESK_SHELL_ARGS="${DESK_SHELL_ARGS:-}"
 
 
 ## Commands
@@ -130,6 +131,9 @@
         exit 1
     else
         local SHELL_EXEC="$(get_running_shell)"
+        if [ -n "$DESK_SHELL_ARGS" ]; then
+            set -- "$DESK_SHELL_ARGS" "$@"
+        fi
         DESK_NAME="${TODESK}" DESK_ENV="${DESKPATH}" exec "${SHELL_EXEC}" "$@"
     fi
 }
@@ -175,7 +179,7 @@
             len=${#name}
             (( len > longest )) && longest=$len
         fi
-    done < <(find "${DESKS}/" -name "*${DESKEXT}" -print0)
+    done < <(find "${DESKS}/" -name "*${DESKEXT}" -print0 | sort -zV)
     if [[ "$AUTO_ALIGN" != false ]]; then
         print_aligned "$out" "$longest"
     else
